.github/workflows/mend.yml

```diff
@@ -0,0 +1,36 @@
+name: mend_scan
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout repo content
+      uses: actions/checkout@v2 # checkout the repository content to github runner.
+      with:
+        fetch-depth: 1
+    - name: setup ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.7
+    - name: create lock
+      run: bundle lock
+    # install java
+    - uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin' # See 'Supported distributions' for available options
+        java-version: '17'
+    # download mend
+    - name: download_mend
+      run: curl -o wss-unified-agent.jar https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
+    - name: run mend
+      run: java -jar wss-unified-agent.jar
+      env:
```

Comment:
we've been setting ```WS_FILESYSTEMSCAN: false``` so that mend doesn't scan the ubuntu machine it's running on and 
```
WS_CHECKPOLICIES: true
WS_FORCEUPDATE: true
```
in anticipation of one day having policies that could be violated and cause a failure. Not sure about the second ones but I think the filesystem one is probably good to have in here.